app/presenters/queries/content_item_presenter.rb

```diff
@@ -86,23 +86,23 @@ def select_fields(scope)
         fields_to_select = fields.map do |field|
           case field
           when :publication_state
-            "#{publication_state_sql} AS publication_state"
+            "#{PUBLICATION_STATE_SQL} AS publication_state"
           when :user_facing_version
             "user_facing_versions.number AS user_facing_version"
           when :lock_version
             "lock_versions.number AS lock_version"
           when :description
             "description->>'value' AS description"
           when :internal_name
-            "#{internal_name_sql} AS internal_name"
+            "#{INTERNAL_NAME_SQL} AS internal_name"
```

Comment:
Frankly the entire branch of the case statement is redundant since we changed the data source for the linkables endpoint, but I didn't want to get into that here.